cpp/src/lance/arrow/type.cc

```diff
@@ -26,12 +26,49 @@
 
 namespace lance::arrow {
 
+namespace {
+
+std::string ToString(::arrow::TimeUnit::type unit) {
+  using TimeUnit = ::arrow::TimeUnit;
+  switch (unit) {
```

Comment:
I forgot, arrow TimeUnit only goes up to 's'?

Reply:
Yes, `TimeUnit` is from nanoseconds to seconds

https://github.com/apache/arrow/blob/master/cpp/src/arrow/type_fwd.h#L225-L231